src/cascadia/TerminalApp/CommandPalette.h

```diff
@@ -40,6 +52,13 @@ namespace winrt::TerminalApp::implementation
         static int _getWeight(const winrt::hstring& searchText, const winrt::hstring& name);
         void _close();
 
+        // TabSwitcher
+        std::optional<winrt::Windows::System::VirtualKey> _anchorKey;
+        bool _tabSwitcherMode{ false };
```

Comment:
nit: future-proof this by making the mode an enum 
```c++
enum class CommandPaletteMode {
	ActionMode=0,
	TabSwitcherMode
}
```

Mainly so I can later re-use this for `CommandlineMode` 😉